docs/documentation/install/requirements.md

```diff
@@ -2,6 +2,7 @@
 
 The kit runs on Mac, Windows and Linux. At a minimum you’ll need Node.js (install instructions below) and a web browser.
 
+If you're using an M1 Mac, you might experience issues when you run the Prototype Kit. To get support, please [contact the Design System team](https://design-system.service.gov.uk/get-in-touch/).
```

Comment:
is it worth something like "(affects Macs bought since 2020)"
https://en.wikipedia.org/wiki/Apple_M1#Products_that_include_the_Apple_M1

Reply:
just wondering if thats a simple thing that might help you know you have an M1

Reply:
That sounds like a good idea. Could we add something like:

> If you're using a M1 Mac (any Mac from after 2020), you might experience issues when you run the kit.

(I wrote 'kit' in lowercase this time, as that's how the remainder of the page refers to it.)

Reply:
hmm might need a nuance - you can still get non M1 macs from 2020 - but you wouldnt have one if its older

Reply:
Ah I see. What about:

> If you're using a M1 Mac (certain Macs from after 2020), you might experience issues when you run the kit.

I'd suggest linking to the wiki article, but I don't know what the org stance is about linking to Wikipedia.

Reply:
I think linking is fine, and would it be 'an M1 Mac'?

Reply:
It would! Thanks Joe - I'll make the change.